atomicapp/providers/lib/kubeconfig.py

```diff
@@ -14,6 +14,66 @@
 class KubeConfig(object):
 
     @staticmethod
+    def from_file(filename):
+        '''
+        Load a file using anymarkup
+
+        Params:
+            filename (str): File location
+        '''
+
+        return anymarkup.parse_file(filename)
+
+    @staticmethod
+    def from_params(api=None, auth=None, ca=None, verify=True):
+        '''
+        Creates a .kube/config based upon the arguments given.
```

Comment:
`Creates a .kube/config` confused me a bit. This doesn't actually create a file, but rather a data structure with the information in it. How about we use something like: `Creates a dict of .kube/config data based upon the arguments given` ?